.kokoro/docs.sh

```diff
@@ -30,4 +30,4 @@ BIN=./node_modules/.bin
 npm install broken-link-checker
 npm install http-server
 $BIN/http-server -p 8080 docs/ &
-$BIN/blc http://localhost:8080 -r --exclude www.gstatic.com --exclude www.googleapis.com
+$BIN/blc http:\/\/localhost:8080 --exclude www.gstatic.com/bigquerydatatransfer -r --exclude www.googleapis.com
```

Comment:
Isn't that just going to get wrecked by autosynth?

Reply:
I made the replacement in the synth script here as well :)